pilot/pkg/xds/endpoints/endpoint_builder.go

```diff
@@ -733,22 +732,11 @@ func supportTunnel(b *EndpointBuilder, e *model.IstioEndpoint) bool {
 // waypointInScope computes whether the endpoint is owned by the waypoint
 func waypointInScope(waypoint *model.Proxy, e *model.IstioEndpoint) bool {
 	scope := waypoint.WaypointScope()
-	if scope.Namespace != e.Namespace {
-		return false
-	}
-	ident, _ := spiffe.ParseIdentity(e.ServiceAccount)
-	if scope.ServiceAccount != "" && (scope.ServiceAccount != ident.ServiceAccount) {
-		return false
-	}
-	return true
+	return scope.Namespace == e.Namespace
```

Comment:
Yeah see prev - "WaypointScope" is just a single string now, I don't see the point in using a type for it if that's all it is.